connect/src/com/telenor/connect/id/ConnectIdService.java

```diff
@@ -58,6 +58,14 @@ public String getAccessToken() {
         return retrieveTokens().getAccessToken();
     }
 
+    public Date getAccessTokenExpirationTime() {
+        if (retrieveTokens() == null) {
```

Comment:
I see what you mean. `retrieveTokens()` uses a cache and only actually fetches the tokens from the store if they are missing. It's a private method.